app.json

```diff
@@ -21,6 +21,9 @@
     "HEROKU_APP_NAME": {
       "required": true
     },
+    "JS_EXCEPTION_LOGGER_KEY": {
```

Comment:
I take it rollbar is smart enough to notice when things are coming from the PR builds and keep those separate from prod logs?

Reply:
The request url in the of the problem will be the PR build. If it becomes difficult to distinguish with just that we can adjust.